frontend/src/components/v-summary-charts.vue

```diff
@@ -316,6 +330,24 @@ export default {
       return repo.location;
     },
 
+    getRepoType(repo) {
+      const repoLink = this.getRepoLink(repo);
+
+      if (repoLink.includes('github')) {
+        return 'github';
+      }
+
+      if (repoLink.includes('gitlab')) {
+        return 'gitlab';
+      }
+
+      if (repoLink.includes('bitbucket')) {
+        return 'bitbucket';
+      }
+
+      return 'default';
+    },
+
```

Comment:
I don't agree with this method of detection. You could have a repo hosted on gitlab with github in its name and this will identify it wrongly. 

The identification is already done by the backend code and can be found in `window.REPOS[repo].location.domainName` which gives the domain type. You can use that instead.

Reply:
Guess I missed that part, will patch it